CONTRIBUTING.md

```diff
@@ -134,13 +134,13 @@ Must be one of the following:
 
 ### Scope
 The scope should be the name of the npm package affected (as perceived by the person reading the changelog 
-generated from commit messages).
+generated from commit messages). Capitalize the first letter.
 
 The following is the list of supported scopes:
 
-* **core**
-* **platform**
-* **doc-app**
+* **Core**
+* **Platform**
+* **Doc-app**
 
```

Comment:
scope `e2e` should be added, yes?
cc: @InnaAtanasova

Reply:
added `E2E` scope